### ISSUE TITLE
After v33 -> v34:  Can not create query syntax schema for property 'tags' 

### ISSUE THREAD

**Issue by tvld** [feathersjs/feathers#2980]
I upgraded from v33 to v34 this morning. But running ``npm run dev`` gives now: ``Error: Can not create query syntax schema for property 'tags'. Only types string, number, integer, boolean, null are allowed.``

This worked ok in v33 
In my schema:
```ts
 tags: Type.Optional(Type.Array(Type.String())),
```
I tried but this did not help:
```ts
export const myQuerySchema = Type.Intersect(
  [
    querySyntax(myQueryProperties, {
      tags: {
        $in: Type.Array(Type.String())
      }
    })
  ]
)
```

**Comment by daffl** [comment]
Sorry about that, this error has been fixed in `5.0.0-pre.35` that I just published. One thing worth noting that `$in` will only work with array **values** in MongoDB.

**Comment by tvld** [comment]
Geat, can confirm fixed in v35
